common/app/metrics/FrontendMetrics.scala

```diff
@@ -128,7 +127,7 @@ final case class DurationMetric(override val name: String, override val metricUn
   }
 
   // Public for tests.
-  def record(dataPoint: DurationDataPoint): Unit = dataPoints.alter(dataPoint :: _)
+  def record(dataPoint: DurationDataPoint): Unit = dataPoints.alter(dataPoint :: dataPoints.get())
```

Comment:
Without this change I was seeing an 'unbound placeholder parameter error' which I think means scala didn't know how to interpret `_`. What I think we want from this function is to append new data points to the list (without this change we just overwrote any existing datapoints, List always had length of 1).

I don't understand why my change introduced this error though. Any ideas?